modules/updating-sno.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster.adoc
+
```

Comment:
Please add the anchor.

Also, you shouldn't start a concept's title with a gerund.